src/main/java/com/programmers/springweekly/domain/voucher/FixedAmountVoucher.java

```diff
@@ -0,0 +1,32 @@
+package com.programmers.springweekly.domain.voucher;
+
+import lombok.AllArgsConstructor;
+
+import java.util.UUID;
+
+@AllArgsConstructor
+public class FixedAmountVoucher implements Voucher{
```

Comment:
항상 정렬을 잊지 말아주세요~!
인텔리제이에서 자동화하는 설정 방법을 노션에 적어뒀으니 참고해보세요
https://www.notion.so/backend-devcourse/intellij-setting-d4317ba83eee4c4db08afa841bc30258

```suggestion
public class FixedAmountVoucher implements Voucher {
```

Reply:
플러그인까지 전부 적용했습니다!! 너무 잘 적어주셔서,, 쉽게 할 수 있었어요!! 이거,, 진짜 신세계군요,, save on action은 저번에 한 줄 알았는데 안되있었더라구요,, ㅎㅎ 최고입니다!! 👍 👍 

적용 커밋: [d9eaa9b](https://github.com/prgrms-be-devcourse/springboot-basic/pull/675/commits/d9eaa9bc394ec37d69c973b8a244fec8e76f683b)